src/Test/TestAIMuriel.java

```diff
@@ -94,9 +127,9 @@ private boolean testBuildMegatank()
   }
 
   /** Confirm that Muriel will build the correct counter, even when there is only one possible counter. */
```

Comment:
Wrong Javadoc. My bad.

Reply:
Not super worried about documentation on tests, to be perfectly honest.

Reply:
sigh